pelican/tests/output/custom/unbelievable.html

```diff
@@ -43,9 +43,9 @@ <h1 class="entry-title">
         </abbr>
 
         <address class="vcard author">
-                By <a class="url fn" href="./author/alexis-metaireau.html">Alexis Métaireau</a>
+                By                         <a class="url fn" href="./author/alexis-metaireau.html">Alexis Métaireau</a>
```

Comment:
You're right. Would renaming the commit to something closer to `tests: fix`? I'm afraid I don't have time to identify the seperate causes/commits.